spec/cocina/models/collection_shared_examples.rb

```diff
@@ -276,7 +276,7 @@
         expect(instance.administrative.hasAdminPolicy).to eq 'druid:mx123cd4567'
 
         tags = instance.administrative.releaseTags
-        expect(tags).to all(be_instance_of Cocina::Models::ReleaseTag)
+        expect(tags).to all(be_instance_of(Cocina::Models::ReleaseTag))
```

Comment:
leaving out these parens must make it somewhat ambiguous based on some failures I had when writing fileset specs ...